src/Core/Managed/Shared/Extensibility/Implementation/JsonSerializer.cs

```diff
@@ -236,9 +236,11 @@ private static void SerializeTelemetryItem(ITelemetry telemetryItem, JsonWriter
                 ExceptionTelemetry exceptionTelemetry = telemetryItem as ExceptionTelemetry;
                 SerializeExceptionTelemetry(exceptionTelemetry, jsonWriter);
             }
+#pragma warning disable CS0618
```

Comment:
I did. I was omitted in this commit but will be there for the next one (will be available very shortly)